File: docs/legacy-csharp/Gestion Trenes y Vagones/NodoTren.cs
internal class NodoTren
{

        private Tren tren;
        private NodoTren siguiente;

        public Tren Tren { get =>  tren; set => tren = value; }
        public NodoTren Siguiente { get => siguiente; set => siguiente = value; }

        public NodoTren(Tren tren) {

            Tren = tren;
            Siguiente = null;

        }


    }
}

File: src/main/java/co/edu/upb/trenes/models/trenes/NodoTren.java
package co.edu.upb.trenes.models.trenes;

public class NodoTren {
    private Tren tren;
    private NodoTren siguiente;

    public NodoTren(Tren tren) {
        this.tren = tren;
    }

    public Tren getTren() { return tren; }
    public NodoTren getSiguiente() { return siguiente; }
    public void setSiguiente(NodoTren siguiente) { this.siguiente = siguiente; }
}
